kontrakter/siktelseTiltale/1.0/siktelseTiltale.schema.json

```diff
@@ -0,0 +1,870 @@
+{
```

Comment:
@calle-777
Fra Calle:
Postnummer: kun tall - alltid 4 tall? Bør være påkrevd?
Hvorfor addreslinjer påkrevd når postnummer og poststed ikke satt
Gir det noen gang mening med kun adresselinjer??
Kan vi skille på norske og utenladske adresser. OneOff?

Reply:
Laget en norsk og en utlandsk adresse med one of.